recipes/gl2ps/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Enable bash strict mode
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+mkdir -p build
+cd build
+
+cmake -LAH \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_BUILD_TYPE=${CMAKE_PLATFORM_FLAGS[@]+"${CMAKE_PLATFORM_FLAGS[@]}"} \
```

Comment:
Who is setting `CMAKE_PLATFORM_FLAGS` ?

Reply:
I got this from HepMC3:

https://github.com/conda-forge/hepmc3-feedstock/blob/c9c43d086ad39bbdfe1a1cbda5f7a6fc82793a50/recipe/build.sh#L14

Which I expect was (wrongly) copied from [root](https://github.com/conda-forge/root-feedstock/blob/master/recipe/build_root-base.sh), which actually does set this.